docs/kind.md

```diff
@@ -35,14 +95,17 @@ usage: kind.sh [[[-cf|--config-file <file>] [-kt|keep-taint] [-ha|--ha-enabled]
 -i6 | --ipv6                 Enable IPv6. DEFAULT: IPv6 Disabled.
 
 ```
-As seen above if you do not specify any options script will assume the default values. 
+Usage Notes 
```

Comment:
nit, maybe you want to add a header here?
### Usage Notes

Reply:
Sounds good!